server.go

```diff
@@ -591,7 +591,18 @@ func (s *Server) HandleTracePacket(packet []byte) {
 		log.WithError(err).Warn("Trace unmarshaling error")
 		return
 	}
-
+	for _, metricPacket := range newSample.Metrics {
+		metric, err := samplers.ParseMetricSSF(metricPacket)
+		if err != nil {
+			log.WithFields(logrus.Fields{
+				logrus.ErrorKey: err,
+				"packet":        metricPacket.String(),
+			}).Warn("Could not parse packet")
+			s.Statsd.Count("packet.error_total", 1, []string{"packet_type:ssf_metric", "reason:parse"}, 1.0)
+			return
+		}
+		s.Workers[metric.Digest%uint32(len(s.Workers))].PacketChan <- *metric
```

Comment:
1) Is this channel buffered?

2) Do we want to buffer it, and/or send on a separate goroutine?

This was less of an issue before, when metrics parsing sent only to metrics workers, and trace parsing sent only to trace workers. But now that we can send to both, we could end up in a situation where metrics load bottlenecks the span load, or vice versa.